src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -318,6 +318,9 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
       remoteReindex: `${ELASTICSEARCH_DOCS}docs-reindex.html#reindex-from-remote`,
       unfreezeApi: `https://www.elastic.co/guide/en/elastic-stack/9.0/release-notes-elasticsearch-9.0.0.html#remove_unfreeze_rest_endpoint`,
       reindexWithPipeline: `${ELASTICSEARCH_DOCS}docs-reindex.html#reindex-with-an-ingest-pipeline`,
+      logsDatastream: `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/${DOC_LINK_VERSION}/logs-data-stream.html`,
+      usingLogsDbIndexModeWithESSecurity: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/detections-logsdb-index-mode-impact.html`,
+      dataStreamReindex: `${ELASTICSEARCH_DOCS}data-stream-reindex-api.html#reindex-data-stream-api-settings`,
```

Comment:
```suggestion
      logsDatastream: `${ELASTIC_DOCS}manage-data/data-store/data-streams/logs-data-stream`,
      usingLogsDbIndexModeWithESSecurity: `${ELASTIC_DOCS}solutions/security/detect-and-alert/using-logsdb-index-mode-with-elastic-security`,
      dataStreamReindex: `${ELASTIC_DOCS}reference/elasticsearch/rest-apis/reindex-data-stream`,
```
Those should be the correct links in the new 9.0+ docs

Reply:
thanks!